Code/GraphMol/RGroupDecomposition/RGroupCore.h

```diff
@@ -36,9 +39,10 @@ struct RCore {
       }
     }
   }
+
   // Return a copy of core where dummy atoms are replaced by
   // the atomic number of the respective matching atom in mol
-  std::unique_ptr<ROMol> replaceCoreDummiesWithMolMatches(
+  std::unique_ptr<ROMol> replaceCoreDummiesWithMolMatchesOriginal(
```

Comment:
Please feel free to remove `replaceCoreDummiesWithMolMatchesOriginal` as your version is much more comprehensive than the one I had originally put in place to fix a problem I had found.

Reply:
Removed.